# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-hal/src/external_memory/mod.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-hal/src/external_memory/mod.rs b/third_party/rust/gfx-hal/src/external_memory/mod.rs
deleted file mode 100644
--- a/third_party/rust/gfx-hal/src/external_memory/mod.rs
+++ /dev/null
@@ -1,19 +0,0 @@
-//! Structures related to the import external memory functionality.
-
-mod errors;
-pub use errors::*;
-
-pub use external_memory::*;
-
-bitflags!(
-    #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-    /// External memory properties.
-    pub struct ExternalMemoryProperties: u32 {
-        /// The memory can be exported using [Device::export_memory][Device::export_memory].
-        const EXPORTABLE = (1 << 0);
-        /// The memory can be imported using [Device::import_external_image][Device::import_external_image] and [Device::import_external_buffer][Device::import_external_buffer].
-        const IMPORTABLE = (1 << 1);
-        /// The memory created using [Device::import_external_image][Device::import_external_image] and [Device::import_external_buffer][Device::import_external_buffer] can be exported using [Device::export_memory][Device::export_memory].
-        const EXPORTABLE_FROM_IMPORTED = (1 << 2);
-    }
-);